modules/custom/openy_popups/config/schema/openy_popups.schema.yml

```diff
@@ -1,4 +1,4 @@
-openy_popups.config:
```

Comment:
We need to update a config for the old users. Please update it in hook_update_N

Reply:
Hello, 
Configuration form submit handler previously used openy_popups.settings

openy/modules/custom/openy_popups/src/Form/SettingsForm.php line 71

$config = \Drupal::service('config.factory')->getEditable('openy_popups.settings');

So only openy_popups.settings store actual setting data. Maybe it was just misspelling. 

I will update hook update to remove not used conf object